corehq/form_processor/models.py

```diff
@@ -990,7 +991,7 @@ class CommCareCaseIndexSQL(PartitionedModel, models.Model, SaveStateMixin):
     RELATIONSHIP_MAP = {v: k for k, v in RELATIONSHIP_CHOICES}
 
     case = models.ForeignKey(
-        'CommCareCaseSQL', to_field='case_id', db_index=False,
+        'CommCareCaseSQL', to_field='case_id',
```

Comment:
it's usually a best practice to put indexes on foreign key fields. did it maybe suggest this just because it's not used heavily right now?